recipe/meta.yaml

```diff
@@ -23,17 +23,14 @@ build:
 
 requirements:
   build:
-    - git
-    - patch       # [unix]
-    - m2-patch    # [win]
-    - python                                 # [build_platform != target_platform]
-    - {{ compiler('c') }}  # [not win]
-    - {{ compiler('rust') }}  # [not win]
+    - python                            # [build_platform != target_platform]
```

Comment:
As we don't build cross-platform you can remove `python                            # [build_platform != target_platform]`